schemas/sample_registration.json

```diff
@@ -0,0 +1,126 @@
+  "fields": [
+    {
+      "name": "program_id",
+      "valueType": "string",
+      "description": "Unique identifier of the ARGO program.",
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "submitter_donor_id",
+      "valueType": "string",
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "restrictions": {
+        "required": true,
+        "regex": "[A-Za-z0-9\\-\\._]{1,64}"
+      }
+    },
+    {
+      "name": "gender",
+      "valueType": "string",
+      "description": "Description of the donor self-reported gender. Gender is described as the assemblage of properties that distinguish people on the basis of their societal roles.",
+      "restrictions": {
+        "required": true,
+        "codeList": [
+          "Male",
+          "Female",
+          "Other"
+        ]
+      }
+    },
+    {
+      "name": "specimen_submitter_id",
+      "valueType": "string",
+      "description": "Unique identifier of the specimen, assigned by the data provider.",
+      "restrictions": {
+        "required": true,
+        "regex": "[A-Za-z0-9\\-\\._]{1,64}"
+      }
+    },
+    {
+      "name": "specimen_tissue_source",
+      "valueType": "string",
+      "description": "Tissue source of the biospecimen.",
+      "restrictions": {
+        "required": true,
+        "codeList": [
+          "Blood derived",
+          "Blood derived - bone marrow",
+          "Blood derived - peripheral blood",
+          "Bone marrow",
+          "Buccal cell",
+          "Lymph node",
+          "Solid tissue",
+          "Plasma",
+          "Serum",
+          "Urine",
+          "Cerebrospinal fluid",
+          "Sputum",
+          "NOS (Not otherwise specified)",
+          "Other",
+          "FFPE",
+          "Pleural effusion",
+          "Mononuclear cells from bone marrow",
+          "Saliva",
+          "Skin"
+        ]
+      }
+    },
+    {
+      "name": "tumour_normal_designation",
+      "valueType": "string",
+      "description": "Description of the kind of specimen that was collected with respect to tumour/normal tissue origin.",
+      "restrictions": {
+        "required": true,
+        "codeList": [
+          "Normal",
+          "Normal - tissue adjacent to primary tumour",
+          "Primary tumour",
+          "Primary tumour - adjacent to normal",
+          "Primary tumour - additional new primary",
+          "Recurrent tumour",
+          "Metastatic tumour",
+          "Metastatic tumour - metastasis local to lymph node",
+          "Metastatic tumour - metastasis to distant location",
+          "Metastatic tumour - additional metastatic",
+          "Xenograft - derived from primary tumour",
+          "Xenograft - derived from tumour cell line",
+          "Cell line - derived from xenograft tissue",
```

Comment:
The convention is that each every term needs to include `tumour` or `normal` in it. This is to make it explicit whether the specimen is tumour or normal. Consider changing this term to `Cell line - derived from xenograft tumour tissue`

Reply:
@junjun-zhang - the current CV list does contain `tumour` and `normal`. 

Please refer to the [table.](https://docs.google.com/spreadsheets/d/1WaeEHnpPr-hpCu4Typp9Qe79gRNe7PPH1kBikOz8Afk/edit?usp=sharing) It shows the following.

Normal
Normal - tissue adjacent to primary tumour
Primary tumour
Primary tumour - adjacent to normal
Primary tumour - additional new primary
Recurrent tumour
Metastatic tumour
Metastatic tumour - metastasis local to lymph node
Metastatic tumour - metastasis to distant location
Metastatic tumour - additional metastatic
Xenograft - derived from primary tumour
Xenograft - derived from tumour cell line
Cell line - derived from xenograft tumour
Cell line - derived from tumour
Cell line - derived from normal

Reply:
OK, it should be just a typo to be correct here.